jerry-ext/handler/handler-gc.c

```diff
@@ -31,6 +31,13 @@ jerryx_handler_gc (const jerry_value_t func_obj_val, /**< function object */
   (void) args_p; /* unused */
   (void) args_cnt; /* unused */
 
-  jerry_gc ();
+  jerry_gc_mode_t mode = JERRY_GC_SEVERITY_LOW;
+
+  if (args_cnt > 0 && jerry_value_to_boolean (args_p[0]))
```

Comment:
Because of this change, neither `args_cnt` nor `args_p` are unused anymore. (See above.)